rpc/jsonrpc_server.go

```diff
@@ -118,3 +120,26 @@ func (j *JSONRPCServer) UnitPrices(
 	reply.UnitPrices = unitPrices
 	return nil
 }
+
+type NameSpacesPriceArgs struct {
+	NameSpaces []string `json:"namespaces"`
+}
+
+type NameSpacesPriceReply struct {
+	Price []uint64 `json:"price"`
+}
+
+func (j *JSONRPCServer) NameSpacesPrice(
```

Comment:
This is really neat so we basically get local fee markets for different namespaces

Reply:
Yup, I'll add a comment for that.